docs/layout.md

```diff
@@ -0,0 +1,26 @@
+## Project layout
+### Module layout[^1]
```

Comment:
skip line between title and text

Reply:
Do you mean a 2 space indentation for Module layout heading? Could not get this...